lib/routes/kaopunews/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '靠谱新闻',
+    url: 'https://kaopu.news',
```

Comment:
```suggestion
    url: 'kaopu.news',
```
> url: The website URL **without protocol** that corresponds

docs: https://docs.rsshub.app/joinus/new-rss/start-code

Reply:
fix